<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoveNode</class>
 <widget class="QDialog" name="RemoveNode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>388</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Remove Note</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This node is going to be removed, but it has internal references. How should this be handled?</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>196</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>182</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="pBtnCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pBtnOk">
     <property name="text">
      <string>Ok</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="rBtnRemoveRecords">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="rBtnReplaceRecords">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="ComboBox" name="comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">widget/combobox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>rBtnReplaceRecords</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>rBtnRemoveRecords</tabstop>
  <tabstop>pBtnOk</tabstop>
  <tabstop>pBtnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pBtnCancel</sender>
   <signal>clicked()</signal>
   <receiver>RemoveNode</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>599</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rBtnReplaceRecords</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
